include/SDL3/SDL_events.h

```diff
@@ -680,7 +681,7 @@ typedef struct SDL_QuitEvent
  */
 typedef struct SDL_UserEvent
 {
-    Uint32 type;        /**< ::SDL_EVENT_USER through ::SDL_EVENT_LAST-1 */
+    SDL_EventType type; /**< ::SDL_EVENT_USER through ::SDL_EVENT_LAST-1 */
```

Comment:
Not sure about the others, but I guess this will be a problem in c++, no?

Reply:
I'm not exactly sure if this is the problem you're considering, but it seems like it shouldn't be an issue: https://stackoverflow.com/questions/33812998/is-it-allowed-for-an-enum-to-have-an-unlisted-value